App.js

```diff
@@ -174,7 +174,11 @@ export default class App extends React.Component {
 
   _loadResourcesAsync = async () => {
     return Promise.all([
-      Asset.loadAsync([require("./src/assets/background/home/bg_home.png")]),
+      Asset.loadAsync([
+        require("./src/assets/background/home/bg_home.png"),
+        require("./src/assets/icons/ic_hide_text.png"),
+        require("./src/assets/icons/ic_show_text.png"),
+      ]),
```

Comment:
we only need to load homeScreen stuff here and possibly background images in the future. Icons are small enough that there wont be a delay and we can save memory.